Descent3/objinfo.cpp

```diff
@@ -60,8 +60,8 @@ const char *Static_object_names[] = {TBL_GENERIC("GuideBot"), TBL_GENERIC("Chaff
 #define NUM_STATIC_OBJECTS (sizeof(Static_object_names) / sizeof(*Static_object_names))
 
 #ifdef EDITOR
-const char *Movement_class_names[] = {"Standing", "Flying", "Rolling", "Walking", "Jumping"};
-const char *Anim_state_names[] = {"Alert",
+extern char *Movement_class_names[] = {"Standing", "Flying", "Rolling", "Walking", "Jumping"};
+extern char *Anim_state_names[] = {"Alert",
```

Comment:
The proper type is `const char* const` but this could totally be made into a `std::array` of `std::string_view`s. Use of the  `extern` keyword here is invalid as `extern` indicates the symbol is declared elsewhere.